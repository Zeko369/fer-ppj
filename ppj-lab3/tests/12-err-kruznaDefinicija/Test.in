<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 n
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       IDN 1 n
      <T_lista>
       $
     <E_lista>
      OP_PLUS 1 +
      <E>
       <T>
        <P>
         BROJ 1 1
        <T_lista>
         $
       <E_lista>
        $
  <lista_naredbi>
   $
