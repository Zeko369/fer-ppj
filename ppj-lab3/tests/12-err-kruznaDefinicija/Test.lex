IDN 1 n
OP_PRIDRUZI 1 =
IDN 1 n
OP_PLUS 1 +
BROJ 1 1
