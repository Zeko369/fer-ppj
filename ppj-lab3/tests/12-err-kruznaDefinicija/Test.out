err 1 n
